src/Terminal.js

```diff
@@ -954,8 +954,38 @@ var Terminal = {
                 }
 				break;
 			case "top":
-				//TODO List each's script RAM usage
-                post("Not yet implemented");
+                if(commandArray.length != 1) {
+                  post("Incorrect usage of top command. Usage: top"); return;
+                }
+                
+                post("Script                    RAM Usage                 Threads");
+                var currRunningScripts = Player.getCurrentServer().runningScripts;
+                for(i = 0; i < currRunningScripts.length; i++) {
+                    //Check if script is on current server
+                    for(j = 0; j < Player.getCurrentServer().scripts.length; j++) {
+                        if(currRunningScripts[i] === Player.getCurrentServer().scripts[j].filename) {
```

Comment:
Won't this check if the script is present on the current server, not whether it's running from the current server?